get-docker.sh

```diff
@@ -0,0 +1,743 @@
+#!/bin/sh
```

Comment:
wow, why do you need it?

Reply:
Unexpected behavior... 
 To be honest I missed the moment when this file was created